performance/run.js

```diff
@@ -0,0 +1,13 @@
+const { execSync } = require('child_process');
+const argv = require('yargs').argv;
+
+if(process.platform === "darwin") {
+    execSync('docker run --rm -v "$(pwd)":/sitespeed.io sitespeedio/sitespeed.io --config=config/config-chrome.json http://host.docker.internal:8080/performance/performance.html', {stdio: 'inherit', cwd: __dirname});
+    execSync('docker run --rm -v "$(pwd)":/sitespeed.io sitespeedio/sitespeed.io --config=config/config-firefox.json http://host.docker.internal:8080/performance/performance.html', {stdio: 'inherit', cwd: __dirname});
+} else if(process.platform === "linux") {
+    execSync('docker run --rm -v "$(pwd)":/sitespeed.io sitespeedio/sitespeed.io --config=config/config-chrome.json --network=host http://localhost:8080/performance/performance.html', {stdio: 'inherit', cwd: __dirname});
+    execSync('docker run --rm -v "$(pwd)":/sitespeed.io sitespeedio/sitespeed.io --config=config/config-firefox.json --network=host http://localhost:8080/performance/performance.html', {stdio: 'inherit', cwd: __dirname});
```

Comment:
Kunnen we dit ook niet beter gewoon via docker-compose doen? Dan is het kiezen op basis van platform niet meer nodig.

Reply:
Eigelijk wel, ik trek het gelijk.

Reply:
Gepushed!